src/Go.CodeAnalysis/Parser/ImportNode.cs

```diff
@@ -38,7 +38,7 @@ public static bool TryParse(Lexer lexer, IList<Error> errors, out ImportNode par
             }
 
             // Single line import (string immediately following import keyword).
-            if (!lexer.IsCorrectLexemeOperatorOrReportError('(', errors))
+            if (!lexer.CurrentLexeme.Extent.Equals("("))
```

Comment:
nit: here we're allocating a string for this comparison. I wonder if we can do this cheaper with `(TryGetSingleChar(out var c) && c == '(')